hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockOutputStream.java

```diff
@@ -761,4 +761,12 @@ private void handleExecutionException(Exception ex) throws IOException {
     adjustBuffersOnException();
     throw getIoException();
   }
+
+  /**
+   * Get the Replication Index.
+   * @return replicationIndex
+   */
+  public int getReplicationIndex() {
```

Comment:
can this be in default scope?

Reply:
The class we are accessing from is not in the same package

Reply:
Git it. This is ok.